cli/cmd/plugin/unmanaged-cluster/cmd/create.go

```diff
@@ -27,6 +27,9 @@ type createUnmanagedOpts struct {
 	portMapping               []string
 	numContPlanes             string
 	numWorkers                string
+	profile                   string
```

Comment:
Perhaps as a stretch goal of this PR, I'd love to see profile accept multiple packages.

From a configuration standpoint, the API would likely be pretty simple, for example:

```yaml
profile:
- name: fluent-bit.community.tanzu.vmware.com
  config: values.yaml
  version: 1.7.5
```

But the complexity likely comes in from the command line flags. I wonder if a formatting convention would work?

```
--profile fluent-bit.community.tanzu.vmware.com:1.7.5:values.yaml,contour.community.tanzu.vmware.com:2.7.5:values2.yaml
```

In this model, `:` optionally separates package name from version and version from config. `,` separates multiple package entries.

The hope would be that most folks would be using the flag simply like:

```
--profile fluent-bit.community.tanzu.vmware.com
```

But we give the more complex flag option as well.